### PR TITLE
Adding Schema.org tags for better discovery

### LINKED ISSUE #353: Improve discoverability with schema.org tags
Follow the dataverse practices for exposing metadata via schema.org tags: https://dataverse.org/blog/latest-dataverse-update-adds-support-schemaorg

### Acceptance Criteria
- [ ] Make a best effort to populate as many of the dataverse recommended schema.org tags as we can
- [ ] If there are places where we don't have the data, make a ticket for that

Example of schema.org tags for a dataset: 

```json
{
    "@context": "http://schema.org",
    "@type": "Dataset",
    "identifier": "http://hdl.handle.net/1902.1/11992",
    "name": "The Standardized World Income Inequality Database",
    "author": [
        {
            "name": "Frederick Solt",
            "affiliation": "University of Iowa"
        }
    ],
    "datePublished": "2009-08-01",
    "dateModified": "2017-10-30",
    "version": "17",
    "description": "Cross-national research on the causes and consequences of income inequality has been hindered by the limitations of existing inequality datasets: greater coverage across countries and over time is available from these sources only at the cost of significantly reduced comparability across observations. The goal of the Standardized World Income Inequality Database (SWIID) is to overcome these limitations. A custom missing-data algorithm was used to standardize the United Nations University's World Income Inequality Database and data from other sources; data collected by the Luxembourg Income Study served as the standard. The SWIID provides comparable Gini indices of gross and net income inequality for 192 countries for as many years as possible from 1960 to the present along with estimates of uncertainty in these statistics. By maximizing comparability for the largest possible sample of countries and years, the SWIID is better suited to broadly cross-national research on income inequality than previously available sources: it offers coverage double that of the next largest income inequality dataset, and its record of comparability is three to eight times better than those of alternate datasets.  In any papers or publications that use the SWIID, authors are asked to cite the article of record for the data set and give the version number as follows: Solt, Frederick. 2016. \"The Standardized World Income Inequality Database.\" Social Science Quarterly 97(5):1267-1281. SWIID Version 6.1, October 2017.",
    "keywords": [
        "Social Sciences",
        "income inequality",
        "income distribution"
    ],
    "citation": [
        "<a href=\"http://onlinelibrary.wiley.com/doi/10.1111/ssqu.12295/abstract\"  target= \"_new\"> Solt, Frederick.  2016.  \"The Standardized World Income Inequality Database.\"  <i>Social Science Quarterly</i> 97(5):1267-1281.</a>"
    ],
    "temporalCoverage": [
        "1960/2016"
    ],
    "schemaVersion": "https://schema.org/version/3.3",
    "license": {
        "@type": "Dataset",
        "text": "CC0",
        "url": "https://creativecommons.org/publicdomain/zero/1.0"
    },
    "includedInDataCatalog": {
        "@type": "DataCatalog",
        "name": "Harvard Dataverse",
        "url": "https://dataverse.harvard.edu"
    },
    "provider": {
        "@type": "Organization",
        "name": "Dataverse"
    }
} 
```

### PR DESCRIPTION
**Pull request by claudiawulee**
Closes #353

### CODE REVIEW

**Comment by claudiawulee** [comment]
Trying into display the citation but not sure how to with the expected parameter

image

File: app/views/layouts/blacklight/base.html.erb [hunk 1]
```diff
@@ -33,6 +33,49 @@
...
+          "license": 
+          {
+            "@type": "Dataset",
+            "text": "",
+            "url": "",
+          },
+          "provider": 
+          {
+              "@type": "Organization",
+              "name": "PDC Discovery"
```
Comment (bess) [inline app/views/layouts/blacklight/base.html.erb:74]: Pretty sure this organization should be "Princeton University," but let's ask PRDS. @matthewjchandler or @sec122 What do you think?

**Review by bess — CHANGES_REQUESTED** [review]
This looks great! I have a couple of requests, though:
1. Can we please get an automated test for this?
2. I'm posting this to the digital_open_data_and_research channel to request feedback on some of the field values, particularly "Organization." I would think we want that to be "Princeton University", but let's ask.

File: app/views/layouts/blacklight/base.html.erb (hunk 1 above)
Comment (matthewjchandler) [reply app/views/layouts/blacklight/base.html.erb:74]: Yes, the organization should be "Princeton University"

**Comment by claudiawulee** [comment]
Hi @bess! Added automated tests and changed the organization to "Princeton University"

File: app/views/shared/_schema_org.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        ],
+        "schemaVersion": "https://schema.org/version/28.0",
+        <%= license_helper(@document.rights_enhanced) %>
+        "provider":
+        {
+            "@type": "Organization",
+            "name": "Princeton University"
+        }
+    }
+    </script>
```
Comment (hectorcorrea) [inline app/views/shared/_schema_org.html.erb:24]: There is also an additional ticket to add _dates_ once we index them (see https://github.com/pulibrary/pdc_discovery/issues/695)

**Review by bess — APPROVED** [review]

**Pull request merged**